### DISCUSSION TITLE
Google TV: Youtube ads not blocked

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [AdguardTeam/AdGuardHome#5301] [Q&A]
I have AD Guard installed on my network. I get no adverts when watching Youtube from my computer. However, when using my GoogleTV, the youtube ads still show. I read somewhere that Google may be hardcoding their DNS into the Google TV. In order to try an force the issue, I have given the Google TV a static IP and set the DNS to point to AD Guard for the primary DNS and the secondary DNS to point to my router (which also uses ADGuard). YouTube adds still appear in YouTube on the GoogleTV. Is there another way anyone knows about that I can try in order to fix this?

**Comment by kBraDrs** [comment]
From my understanding, YouTube now embed their adverts within the same DNS container as the videos yes. I do not believe there is a way to block these adverts other than with an extension or other adblocking software. 
So you are correct when you say that Google are now preventing this method of blocking their ads and does not appear to be possible at all through the DNS. 
Which would be nice, but I understand their reasoning. If anyone else knows how to block the ads via DNS then I would love to hear also.
